source/examples/qtexample/main.cpp

```diff
@@ -127,6 +127,10 @@ class Window : public WindowQt
         case Qt::Key_F5:
             m_vertexShaderSource->reload();
             m_fragmentShaderSource->reload();
+            updateGL();
+            break;
+        case Qt::Key_Escape:
```

Comment:
other glfw-based sample apps have the 'esc to exit' behavior, mirroring it here for consistency.